torch/csrc/distributed/rpc/script_remote_call.cpp

```diff
@@ -47,9 +47,9 @@ std::unique_ptr<ScriptRemoteCall> ScriptRemoteCall::fromMessage(
   auto retRRefId = ForkId::fromIValue(values.back());
   values.pop_back();
 
-  auto op = ScriptCall::fromIValues(values);
+  auto scriptCallPtr = ScriptCall::fromIValues(values);
   return c10::guts::make_unique<ScriptRemoteCall>(
-      op, std::move(values), std::move(retRRefId), std::move(retForkId));
+      scriptCallPtr->op(), std::move(values), retRRefId, retForkId);
```

Comment:
GlobalUniqueIds should use move semantics to avoid extra copying? why we remove here?

Reply:
ah, possibly an accidental change, will change it back